packages/hyperion-docs/docs/intro.md

```diff
@@ -2,46 +2,48 @@
 sidebar_position: 1
 ---
 
-# Tutorial Intro
+# Hyperion Introduction
 
-Let's discover **Docusaurus in less than 5 minutes**.
+Hyperion is thin and high performance layer that enables
+adding [aspects](https://en.wikipedia.org/wiki/Aspect-oriented_programming) to JavaScript.
 
-## Getting Started
-
-Get started by **creating a new site**.
+With aspects, we can avoid littering the main application code
+with auxilary logic and you can keep all of the related logic
+of a particular aspect in one place.
 
-Or **try Docusaurus immediately** with **[docusaurus.new](https://docusaurus.new)**.
+Hyperion enables **intercepting function, class methods, and attribute accesses on object**. This interception enables unparallel level of "observability" and "control" for aspects.
 
-### What you'll need
+Hyperion's interception is designed carefully to be JIT friendly with very little performance overhead. The module setup is "tree-shaking" friendly so that you only endup with code that your application really need. Hyperion works well with deep-prototype chains on classes and can eaisly handle inconsistent prototype chains of DOM Classes among various browser implementations. 
 
-- [Node.js](https://nodejs.org/en/download/) version 16.14 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
+Hyperion source code is devided into separate packages to make code dependencies explicit and corresponding modules very composable. Please refer to the packages section to see the purpose of each subcomponents of Hyperion.
 
-## Generate a new site
-
-Generate a new Docusaurus site using the **classic template**.
+## Getting Started
 
-The classic template will automatically be added to your project after you run the command:
+First install hyperion from npm
 
 ```bash
-npm init docusaurus@latest my-website classic
+npm install hyperionj
```

Comment:
Should be hyperionjs?

Reply:
Nope, that is intentional! I could not register `hyperionjs` and it was already taken. So, for now I got the `hyperionj` and should see what happens when we upload to it.

I'll land this, but will not yet upload to the website until we get to a better shape and also have a npm package landed.